airflow/providers/amazon/provider.yaml

```diff
@@ -73,9 +73,10 @@ dependencies:
   - mypy-boto3-rds>=1.24.0
   - mypy-boto3-redshift-data>=1.24.0
   - mypy-boto3-appflow>=1.24.0
+  - aiohttp
```

Comment:
where we are using this?

Reply:
removed it: https://github.com/apache/airflow/pull/31018/commits/38583a72a1cf814e5fbdc8000fbe8c21a32f024e